src/api/core/warren/indicators/mixins.py

```diff
@@ -62,20 +56,21 @@ def cache_key(self) -> str:
         ).hexdigest()
         return f"{self.__class__.__name__.lower()}-{lrs_query_hash}"
 
-    async def get_cache(self) -> CacheEntry:
+    async def get_cache(self, session: Session) -> CacheEntry:
```

Comment:
I think `session` is not explicit in the indicator context. WDYT?

```suggestion
    async def get_cache(self, db_session: Session) -> CacheEntry:
```